### PR TITLE
Marina Kuzmenko added homework 1-3

### PR DESCRIPTION
**Pull request by MarinaKuzmenko**

### CODE REVIEW

File: mkuzmenko/HW1/Task1.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = input("What is your name?")
```
Comment (kolyasalubov) [inline mkuzmenko/HW1/Task1.py:1]: good job

File: mkuzmenko/HW1/Task2.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name = input("What is your name?")
```
Comment (kolyasalubov) [inline mkuzmenko/HW1/Task2.py:1]: good job

File: mkuzmenko/HW3/Task1.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+In the face of ambiguity, refuse the temptation to guess.
+There should be one-- and preferably only one --obvious way to do it.
+Although that way may not be obvious at first unless you're Dutch.
+Now is better than never.
+Although never is often better than *right* now.
+If the implementation is hard to explain, it's a bad idea.
+If the implementation is easy to explain, it may be a good idea.
+Namespaces are one honking great idea -- let's do more of those!"""
+
+
```
Comment (kolyasalubov) [inline mkuzmenko/HW3/Task1.py:22]: good job

File: mkuzmenko/HW3/Task2.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+number = input("Enter a four-digit number: ")
```
Comment (kolyasalubov) [inline mkuzmenko/HW3/Task2.py:1]: good job

File: mkuzmenko/HW3/Task3.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+a = int(input('Enter a: '))
```
Comment (kolyasalubov) [inline mkuzmenko/HW3/Task3.py:1]: good job

File: mkuzmenko/HW4/Task1.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+celsius = float(input("Enter the temperature in Celsius: "))
```
Comment (kolyasalubov) [inline mkuzmenko/HW4/Task1.py:1]: good job

File: mkuzmenko/HW1/Task1.py (hunk 1 above)
Comment (MarinaKuzmenko) [reply mkuzmenko/HW1/Task1.py:1]: Thank you

File: mkuzmenko/HW3/Task1.py (hunk 3 above)
Comment (MarinaKuzmenko) [reply mkuzmenko/HW3/Task1.py:22]: Thank you

File: mkuzmenko/HW3/Task2.py (hunk 4 above)
Comment (MarinaKuzmenko) [reply mkuzmenko/HW3/Task2.py:1]: Thank you

File: mkuzmenko/HW3/Task3.py (hunk 5 above)
Comment (MarinaKuzmenko) [reply mkuzmenko/HW3/Task3.py:1]: Thank you

File: mkuzmenko/HW4/Task1.py (hunk 6 above)
Comment (MarinaKuzmenko) [reply mkuzmenko/HW4/Task1.py:1]: Thank you

File: mkuzmenko/HW1/Task2.py (hunk 2 above)
Comment (MarinaKuzmenko) [reply mkuzmenko/HW1/Task2.py:1]: Thank you

**Pull request merged**